src/rtp/ext.rs

```diff
@@ -438,6 +438,22 @@ impl ExtensionMap {
         }
     }
 
+    pub(crate) fn remove(&mut self, ext: Extension) {
+        let maybe_entry = self.0.iter_mut().find_map(|e| {
+            if e.as_mut().map(|e| e.ext == ext).unwrap_or(false) {
+                Some(e)
+            } else {
+                None
+            }
+        });
+
+        let Some(entry) = maybe_entry else {
+            return;
+        };
+
+        let _ = entry.take();
+    }
+
```

Comment:
I don't understand the need for this code block. I thought we already set the a=ssrc from the SDP?